Classes/Core/ConsoleBootstrap.php

```diff
@@ -355,6 +355,7 @@ public function disableCoreCaches()
     public function initializeConfigurationManagement()
     {
         $this->populateLocalConfiguration();
+        $this->initializeRuntimeActivatedPackagesFromConfiguration();
```

Comment:
this must no executed in composer context

Please wrap around a check for composer mode like this:

```php
        if (self::usesComposerClassLoading()) {
            $this->initializeRuntimeActivatedPackagesFromConfiguration();
        }
```